laravel/database/migrations/2021_04_07_162559_rename_probability_and_rotations_columns.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class RenameProbabilityAndRotationsColumns extends Migration
```

Comment:
imo, this should be two separate migrations, since it's modifying two separate tables.